psi4/driver/procrouting/dft_funcs/hyb_superfuncs.py

```diff
@@ -239,7 +232,6 @@ def build_hf3c_superfunctional(name, npoints, deriv, restricted):
     sup.allocate()
     return (sup, ('HF3C', '-d3bj'))
 
-
```

Comment:
There should be two spaces between these functions. Can you format your files with YAPF (`pip install yapf`)?